flowmachine/flowmachine/features/subscriber/contact_balance.py

```diff
@@ -69,59 +67,67 @@ def __init__(
         stop,
         *,
         hours="all",
-        table="all",
+        tables="all",
         subscriber_identifier="msisdn",
         direction="both",
         exclude_self_calls=True,
         subscriber_subset=None,
     ):
-        """
-        """
-        logger.warning(
-            "The the ContactBalance() feature uses CDRs "
-            + "IDs for calculating an subscriber's graph. "
-            + "If IDs are not generated correctly, this "
-            + "feature could yield erroneous results."
-        )
-
-        self.table = table
+        self.tables = tables
         self.start = start
         self.stop = stop
         self.hours = hours
-        self.subscriber_identifier = subscriber_identifier
         self.direction = direction
+        self.subscriber_identifier = subscriber_identifier
         self.exclude_self_calls = exclude_self_calls
 
-        if self.direction not in ("both", "in", "out"):
-            raise ValueError("Unidentified direction: {}".format(self.direction))
+        if self.direction == "both":
+            column_list = [self.subscriber_identifier, "msisdn_counterpart"]
+            self.tables = tables
+        else:
```

Comment:
Why was the check removed that `direction` needs to be one of `("both", "in", "out")`? With the new logic I could be passing arbitrary values for `direction`, no?